Doc/library/importlib.metadata.rst

```diff
@@ -189,9 +215,16 @@ for more information on entry points, their definition, and usage.
 Distribution metadata
 ---------------------
 
-Every `Distribution Package <https://packaging.python.org/en/latest/glossary/#term-Distribution-Package>`_ includes some metadata,
-which you can extract using the
-``metadata()`` function::
+.. function:: metadata(distribution_name)
+
+   Return the distribution metadata corresponding to the named
+   distribution package.
+
+   Raises :class:`PackageNotFoundError` if the named distribution
+   package is not installed in the current Python environment.
+
+Every `Distribution Package <https://packaging.python.org/en/latest/glossary/#term-Distribution-Package>`_
+includes some metadata, which you can extract using the ``metadata()`` function::
```

Comment:
```suggestion
includes some metadata, which you can extract using the :func:`!metadata` function::
```

Reply:
Applied locally (along with many other similar changes to use semantic cross-references internally).